### PR TITLE
Fix premierefc

### PR DESCRIPTION
**Pull request by opssemnik**
Fix the already dark main game showcase (div), its text (h1).
Do not invert game images

### CODE REVIEW

**Comment by opssemnik** [comment]
Oh and by the way, the URL has to be like that, the main domain (globosatplay.globo.com) is used for all ondemand services that "GLOBO" licenses here in brazil (which includes both international channels like fox and universal services) and local channels (in this case ,premierefc is for Brazillian football with some LATAM games)

**Comment by opssemnik** [comment]
I also added universal play to this patch, to keep alphabetical order

File: src/config/fix_inversion.json [hunk 1]
```diff
@@ -224,6 +224,15 @@
                 ".ui_light .navbar"
             ]
         },
+        {
+            "url": "globosatplay.globo.com/premierefc/ao-vivo",
```
Comment (alexanderby) [inline src/config/fix_inversion.json]: I think you can merge these 2 fixes into 1 `globosatplay.globo.com`

File: src/config/fix_inversion.json [hunk 2]
```diff
@@ -224,6 +224,15 @@
...
+            "invert": "h1, div#ao-vivo.palco.pre-evento.pfc",
```
Comment (alexanderby) [inline src/config/fix_inversion.json]: `#ao-vivo.palco` should be enough. Have a look if `h1` element has some class.

File: src/config/fix_inversion.json [hunk 3]
```diff
@@ -224,6 +224,15 @@
...
+            "noinvert": "img"
+        },
+        {
+            "url": "globosatplay.globo.com/universal/",
+            "invert": "a, header.rmq-e9c50d2d, div.rmq-26e388b"
```
Comment (alexanderby) [inline src/config/fix_inversion.json]: `a` selector is too generic. `header` and `div` can be removed

**Comment by alexanderby** [comment]
For making changes you can edit existing Pull Request

File: src/config/fix_inversion.json (hunk 1 above)
Comment (alexanderby) [reply src/config/fix_inversion.json]: I've re-read about the URL case, let it be 2 separate.

**Comment by alexanderby** [comment]
Many changes and improvements were made since the PR was opened, please, submit new PR if changes are still needed.

**Pull request closed without merging**